fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductReviewLoading.kt

```diff
@@ -0,0 +1,88 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.shopping.ui
+
+import androidx.compose.animation.core.LinearEasing
+import androidx.compose.animation.core.RepeatMode
+import androidx.compose.animation.core.StartOffset
+import androidx.compose.animation.core.animateFloat
+import androidx.compose.animation.core.infiniteRepeatable
+import androidx.compose.animation.core.rememberInfiniteTransition
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.theme.FirefoxTheme
+
+private const val ANIMATION_DURATION_MS = 1000
+private const val NUMBER_OF_BOXES = 5
+private const val INITIAL_ALPHA = 0.25f
+private const val TARGET_ALPHA = 1f
+private val boxHeights = listOf(80.dp, 80.dp, 192.dp, 40.dp, 192.dp)
+private val animationInitialOffsetMillis = listOf(500, 1000, 0, 500, 1000)
+
+/**
+ * Loading UI for review quality check content.
+ */
+@Composable
+fun ProductReviewLoading() {
+    val infiniteTransition = rememberInfiniteTransition("ProductReviewLoading")
+
+    Column(
+        modifier = Modifier.padding(0.dp),
```

Comment:
nit: think we can remove setting the padding

Reply:
good catch, updated and added modifier default param for `ProductReviewLoading`.